source/fundamentals/indexes.txt

```diff
@@ -397,6 +397,34 @@ Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB s
 
 .. driver-content-end
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations on **clustered collections**.
+Clustered collections store documents ordered by the clustered index key
+value.
+
+To create a clustered index, specify a ``ClusteredIndexOptions`` in
+your ``CreateCollectionOptions``. The ``ClusteredIndexOptions`` must
+specify the ``_id`` field as the key and the unique field as ``true``.
+
+The following example creates a clustered index on the ``_id`` field in
+the ``tea.vendors`` collection:
+
+.. code-block:: java
+   
+   MongoDatabase database = mongoClient.getDatabase("tea");
+   ClusteredIndexOptions clusteredIndexOptions = new ClusteredIndexOptions(Filters.eq("_id", 1), true);
+   CreateCollectionOptions createCollectionOptions = new CreateCollectionOptions().clusteredIndexOptions(clusteredIndexOptions);
+
+   database.createCollection("vendors", createCollectionOptions);
+
+See the MongoDB server manual sections on 
+:ref:`Clustered Indexes <db.createCollection.clusteredIndex>` and 
+:ref:`Clustered Collections <clustered-collections>`
+for more information.
```

Comment:
Suggestion: for consistency with other sections, when presenting multiple links, use a bulleted list.